src/content/docs/pt-br/pages/devtools/cli/azion-cli-alpha/edge-application/edge-application.mdx

```diff
@@ -0,0 +1,250 @@
+---
+title: Azion CLI edge-application
+description: a
+meta_tags: CLI
+namespace: documentation_cli_edge_application
+menu_namespace: cliMenuAlpha
+permalink: /documentacao/produtos/cli/edge-application/
+---
+
+## Criar
+
+### Uso
+
+```bash
+$ azion create edge-application --name "naruno"
+```
+
+### Flags obrigatórias
+
+#### name
+
+A flag `--name` define o nome da edge application que está sendo criada. É obrigatória se a flag `--in` não for usada.
+
+### Flags opcionais
+
+#### address
+
+A opção `--address` especifica o endereço de um recurso ou servidor.
+
+#### browser-cache-settings
+
+A opção `--browser-cache-settings` configura a quantidade de tempo que o conteúdo é armazenado em cache no navegador do usuário. Os valores possíveis são `honor` ou `override`.
+
+#### browser-cache-settings-maximum-ttl
+
+A opção `--browser-cache-settings-maximum-ttl` define o tempo máximo de vida (TTL) dos recursos em cache no navegador. Ela estabelece um limite de tempo para quanto tempo os recursos podem ser armazenados em cache no navegador.
+
+#### cdn-cache-settings
+
+A opção `--cdn-cache-settings` configura como a Azion armazena em cache o conteúdo no edge. Os valores possíveis são `honor` ou `override`.
+
+#### cdn-cache-settings-maximum-ttl
+
+A opção `--cdn-cache-settings-maximum-ttl` define o tempo máximo de vida (TTL) dos recursos em cache no edge. Ela estabelece um limite de tempo para quanto tempo os recursos podem ser armazenados em cache nos edge nodes.
+
+#### delivery-protocol
+
+A opção `--delivery-protocol` especifica se os dados devem ser entregues via HTTP ou HTTPS.
+
+#### host-header
+
+A opção `--host-header` permite a personalização do cabeçalho `Host`.
+
+#### http-port
+
+A flag `--http-port` especifica a porta de entrega HTTP da edge application. O valor padrão é `80`.
+
+#### https-port
+
+A flag `--https-port` especifica a porta de entrega HTTPS da edge application. O valor padrão é `443`.
+
+#### http3
+
+A opção `--http3` permite a entrega em HTTP/3.
+
+#### in
+
+A opção `--in` especifica o caminho para um arquivo JSON que contém os atributos da edge application sendo criada. Também pode ser lido a partir do stdin usando `-`.
```

Comment:
o caminho do arquivo pode ser lido ou a flag in?

Reply:
```suggestion
A opção `--in` especifica o caminho para um arquivo JSON que contém os atributos da edge application sendo criada. O caminho também pode ser lido a partir do stdin usando `-`.
```